src/rust_wgpu_backend/codegen.rs

```diff
@@ -538,7 +538,13 @@ impl<'program> CodeGen<'program>
 
 				use std::iter::FromIterator;
 
-				let argument_var_ids = Vec::from_iter(arguments.iter().enumerate().map(|(index, x)| { let slot_id = input_slot_ids[index]; placement_state.slot_variable_ids[& slot_id] })).into_boxed_slice();
+				dbg!(function);
```

Comment:
I think a few surgical devices are being left in the patient here

Reply:
oh, uh, yeah, oops.  I'm not actually sure how that got there